orion-server/src/main/java/com/pinterest/orion/core/actions/memq/MemqClusterDecommissionBrokers.java

```diff
@@ -0,0 +1,10 @@
+package com.pinterest.orion.core.actions.memq;
+
+import com.pinterest.orion.core.actions.Action;
+import com.pinterest.orion.core.actions.generic.GenericClusterWideAction;
+
+public abstract class MemqClusterDecommissionBrokers extends GenericClusterWideAction.ParallelDecommissionNodeAction {
```

Comment:
This abstract class is not used anywhere; do we need a basic implementation of it? Same for below two classes.

Reply:
This class will be inheritant to run multiple MemqBrokerDecommissionActions in parallel (ClusterDecomission->BrokerDecomission). I will add more details in javadoc.